geth/jail/internal/loop/loop_test.go

```diff
@@ -53,7 +53,7 @@ func (s *LoopSuite) SetupTest() {
 
 	ctx, cancel := context.WithCancel(context.Background())
 	s.cancel = cancel
-	go s.loop.Run(ctx)
+	go s.loop.Run(ctx) //nolint: errcheck
```

Comment:
it will be useful to know an error if loop will crush

Reply:
I'm not 100% familiar with these tests, but it looks like in all of these tests we're canceling the context at the end, which will then always returns a `context.Canceled` error here. I've added a check - please let me know if this isn't a reasonable assumption.